features/integration/abi.feature

```diff
@@ -0,0 +1,48 @@
+@abi
+Feature: ABI Interaction
+  Background:
+    Given an algod v2 client
+    And a signing account with address "BH55E5RMBD4GYWXGX5W5PJ5JAHPGM5OXKDQH5DC4O2MGI7NW4H6VOE4CP4" and mnemonic "awful drop leaf tennis indoor begin mandate discover uncle seven only coil atom any hospital uncover make any climb actor armed measure need above hundred"
+
+  Scenario: AtomicTransactionComposer test
+    Given a new AtomicTransactionComposer
+    And suggested transaction parameters fee 1234, flat-fee "true", first-valid 9000, last-valid 9010, genesis-hash "Mf0h6zjkEIEZPtNM3zsrg+iHQFS0fZxhgr7w35I464M=", genesis-id "cucumbernet"
```

Comment:
I realized this won't work on an integration test anyway, since it uses hardcoded values. Probably we'll need a step like `And suggested transaction parameters from the algod v2 client` to get the values from algod